doc/source/debian.rst

```diff
@@ -0,0 +1,177 @@
+
+
+Environment
+-----------
+
+We need some packages to compile the package
+
+::
+
+	# dpkg-buildpackage and mk-build-deps will be used
+	user@debian:~$ sudo apt install dpkg-dev devscripts
+	[...]
+
+	# the build directory is where the final package files will be
+	user@debian:~$ mkdir build
+	user@debian:~$ cd build
+
+	# clone the snuffleupagus repo
+	user@debian:~/build$ git clone https://github.com/nbs-system/snuffleupagus.git
+	Cloning into 'snuffleupagus'...
+	remote: Counting objects: 3003, done.
+	remote: Compressing objects: 100% (6/6), done.
+	remote: Total 3003 (delta 1), reused 2 (delta 0), pack-reused 2997
+	Receiving objects: 100% (3003/3003), 9.19 MiB | 7.41 MiB/s, done.
+	Resolving deltas: 100% (2032/2032), done.
+	user@debian:~/build$ cd snuffleupagus/
+	user@debian:~/build/snuffleupagus$ 
+
+	user@debian:~/build/snuffleupagus$ mk-build-deps 
+	Use of uninitialized value $fullname in concatenation (.) or string at /usr/bin/equivs-build line 210.
+	dh_testdir
+	dh_testroot
+	dh_prep
+	dh_testdir
+	dh_testroot
+	dh_install
+	dh_install: Compatibility levels before 9 are deprecated (level 7 in use)
+	dh_installdocs
+	dh_installdocs: Compatibility levels before 9 are deprecated (level 7 in use)
+	dh_installchangelogs
+	dh_installchangelogs: Compatibility levels before 9 are deprecated (level 7 in use)
+	dh_compress
+	dh_fixperms
+	dh_installdeb
+	dh_installdeb: Compatibility levels before 9 are deprecated (level 7 in use)
+	dh_gencontrol
+	dh_md5sums
+	dh_builddeb
+	dpkg-deb: building package 'snuffleupagus-build-deps' in '../snuffleupagus-build-deps_0.3.0-1_all.deb'.
+
+	The package has been created.
+	Attention, the package has been created in the current directory,
+	not in ".." as indicated by the message above!
+
+	# install the dependencies
+	# you can install php7.{0,1,2}-dev packages, the resulting snuffleupagus package will be built against each of these php7 versions
+	user@debian:~/build/snuffleupagus$ sudo dpkg -i snuffleupagus-build-deps_0.3.0-1_all.deb 
+	Selecting previously unselected package snuffleupagus-build-deps.
+	(Reading database ... 29348 files and directories currently installed.)
+	Preparing to unpack snuffleupagus-build-deps_0.3.0-1_all.deb ...
+	Unpacking snuffleupagus-build-deps (0.3.0-1) ...
+	dpkg: dependency problems prevent configuration of snuffleupagus-build-deps:
+	 snuffleupagus-build-deps depends on php7.0-dev | php7.1-dev | php7.2-dev; however:
+	  Package php7.0-dev is not installed.
+	  Package php7.1-dev is not installed.
+	  Package php7.2-dev is not installed.
+
+	dpkg: error processing package snuffleupagus-build-deps (--install):
+	 dependency problems - leaving unconfigured
+	Errors were encountered while processing:
+	 snuffleupagus-build-deps
+
+	user@debian:~/build/snuffleupagus$ sudo apt --fix-broken install
+	[...]
+
+Compilation
+-----------
+
+Now we can compile the package
+
+::
+
+	# build the package
+	# DEB_BUILD_OPTIONS="noddebs nocheck"
```

Comment:
- Why the `nocheck`?
- I disagree with `noddebs`: having one will vastly easy the bug-reporting process.

Reply:
I agree that these DEB_BUILD_OPTIONS suit my needs but it shouldn't be in the official documentation